spec/takeaway_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'takeaway'
+
+describe Menu do
+  it 'shows list of dishes' do
+  end
```

Comment:
Did you forget to write this test, because it showed as 'passing'? If you run a
```ruby
it "blah blah" do
end
```
with nothing in it, it will show up as passing. You can tag it as pending by using `xit` instead of `it`.